document-store/src/main/java/org/hypertrace/core/documentstore/Collection.java

```diff
@@ -36,6 +36,13 @@
    */
   boolean updateSubDoc(Key key, String subDocPath, Document subDocument);
 
+  /**
```

Comment:
Can you please update the Java Docs, explaining what the function does and what does the argument mean?